src/AppEntry.tsx

```diff
@@ -74,13 +73,30 @@ const AppWithConfig: React.FunctionComponent = () => {
 // eslint-disable-next-line react/display-name
 const AppEntry: React.FunctionComponent = React.memo(() => (
   <Provider store={init(logger).getStore()}>
-    <I18nextProvider i18n={appServicesi18n}>
+    <I18nProvider
+      lng={'en'}
+      resources={{
+        en: {
+          common: () => import('@rhoas/app-services-ui-components/locales/en/common.json'),
+          'create-kafka-instance': () =>
+            import('@rhoas/app-services-ui-components/locales/en/create-kafka-instance.json'),
+          kafka: () => import('@rhoas/app-services-ui-components/locales/en/kafka.json'),
+          metrics: () => import('@rhoas/app-services-ui-components/locales/en/metrics.json'),
+          // temporary translations until all user facing visuals are ported to the ui components repo
+          appUi: () => import('./locales/app-services-ui.json'),
```

Comment:
```suggestion
          appUI: () => import('./locales/app-services-ui.json'),
```
or may be we don't need postfix `UI`

Reply:
Changed all the temporary keys to make it super clear we have to fix this